src/form-elements/FormElementPointCadastralParcel.tsx

```diff
@@ -0,0 +1,229 @@
+          })
+        }
+      } catch (err) {
+        if (!abortSignal.aborted) {
+          setState({
+            isLoading: false,
+            error: err as Error,
+          })
+        }
+      }
+    },
+    [element, formId, label, onChange, parcelId],
+  )
+
+  // Ensure the label is set if the value is set outside of this component
+  React.useEffect(() => {
+    if (value) {
+      if (!parcelId) {
+        // Record in value parcelId. This should never happen
+        return
+      }
+      if (label !== parcelId) {
+        setLabel(parcelId)
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [value])
+
+  const { isLookingUp } = React.useContext(LookupNotificationContext)
+  const isDisplayingValidationMessage =
+    (((isDirty || displayValidationMessage) &&
+      !!validationMessage &&
+      !isLoading) ||
+      error) &&
+    !isLookingUp
+
+  const hasCopyButton = !!value && !!element.readOnly
+  return (
+    <div className="cypress-point-cadastral-parcel-element">
+      <FormElementLabelContainer
+        className={pointCadastralParcelClass}
+        id={id}
+        element={element}
+        required={element.required}
+      >
+        <div
+          className={clsx('field has-addons', {
+            'no-addons-mobile': !hasCopyButton,
+          })}
+        >
+          <div
+            className={clsx('control is-expanded has-icons-right', {
+              'is-loading': isLoading,
+              'has-icons-right': !isLoading,
+            })}
+          >
+            <input
+              type="text"
+              id={id}
+              name={element.name}
+              className="input ob-input cypress-point-cadastral-parcel-control"
+              placeholder={element.placeholderValue}
+              value={label}
+              onChange={(e) => {
+                setLabel(e.target.value)
+                if (value) {
+                  onChange(element, {
+                    value: undefined,
+                  })
+                }
+              }}
+              required={element.required}
+              disabled={element.readOnly || isLoading}
+              onBlur={() => {
+                setIsDirty()
+                loadCadastralParcel(new AbortController().signal)
+              }}
+              aria-describedby={ariaDescribedby}
+              autoComplete={autocompleteAttributes}
+            />
+            {!isLoading && (
+              <span className=" ob-input-icon icon is-small is-right">
+                {value ? (
+                  <MaterialIcon className="is-size-5 has-text-success">
+                    check
+                  </MaterialIcon>
+                ) : error?.message ? (
+                  <MaterialIcon className="is-size-5 has-text-danger">
+                    error
+                  </MaterialIcon>
+                ) : (
+                  <MaterialIcon className="is-size-5">error</MaterialIcon>
+                )}
```

Comment:
why two very slightly different renders here?

Reply:
that should be `map`, I must not have committed it.